extensions/ql-vscode/src/vscode-tests/test-config.ts

```diff
@@ -79,7 +82,10 @@ export const testConfigHelper = async (mocha: Mocha) => {
     },
     async afterAll() {
       // Restore all settings to their default values after each test suite
-      await Promise.all(TEST_SETTINGS.map(setting => setting.restoreToInitialValues()));
+      // Only do this outside of CI since the sometimes hangs on CI.
```

Comment:
Why does this hang in the `afterAll` but not in the `beforeEach`?

Reply:
I'm not exactly sure why. It looks like by the time the `afterAll` the workspace is already shut down and errors are thrown when you try to write settings. This seems to have been causing sporadic issues with the cli-integration tests on windows. The error stack traces are helpfully(!) hidden unless you run vscode with verbose logging.